projects/WeTek_Core/initramfs/platform_init

```diff
@@ -1,20 +1,20 @@
 #!/bin/sh
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2014 Alex Deryskyba (alex@codesnake.com)
```

Comment:
@stefansaraev I wasn't voting for dropping any copyright, I read it as codesnake said he hasn't done there something (my fault) therefor would it be unnecessary to include any copyright for him (tx to clarify that). As said we have problems to claim who has done what and therefore we keep the existing copyright (looks like that drop was overseen in review).